### ISSUE TITLE
Wrong typescript re-export in index.d.ts

### ISSUE THREAD

**Issue by stalniy** [crcn/sift.js#178]
Inside [`index.d.ts`](https://github.com/crcn/sift.js/blob/master/index.d.ts) file there is incorrect re-export . Only `lib/core` is exported but it should 

```ts
export * from './lib'
```

The workaround is to define module with `declare`:

```ts
declare module 'sift' {
  import sift from 'sift/lib';
  export default sift;
  export * from 'sift/lib'
}
```

**Comment by crcn** [comment]
Published `11.1.2` to NPM with a fix. Lmk if the problem is solved on your end. 👍

**Comment by stalniy** [comment]
No, it doesn't work. `tsc` shows `Found 44 errors.`

If I change `index.d.ts` to this:

```ts
import sift from './lib';

export default sift;
export * from "./lib";
```

all works as expected.

I think you shouldn't reexport typescript files from `d.ts` file (`export from './src'` does this). That's why you need to reexport from `./lib`

**Comment by sandersn** [comment]
@stalniy You mean `export * from './lib'` is correct, right?

**Comment by stalniy** [comment]
`export * ` does not export default exported symbol that’s why you need additional import

**Comment by crcn** [comment]
Fix has been published as `11.1.4`, let me know if it works.

**Comment by stalniy** [comment]
Updated dependency and all good. Thanks!
